src/main/java/net/fabricmc/loader/api/metadata/CustomValue.java

```diff
@@ -91,6 +112,20 @@
 		 * @return the value associated, or {@code null} if no such value is present
 		 */
 		CustomValue get(String key);
+
+		/**
+		 * Returns a sequential {@link Stream} with this iterable as its source.
+		 */
+		default Stream<Map.Entry<String, CustomValue>> stream() {
+			return StreamSupport.stream(this.spliterator(), false);
```

Comment:
This may be a bit redundancy since this is already Iterable? I am not against it tbf

Reply:
`stream` is in `Collection`, not `Iterable`

Reply:
Yeah I realized that. It's fine imo